### ISSUE TITLE
PRIu64

### ISSUE THREAD

**Issue by 3oheicrw** [LADSoft/OrangeC#717]
Source: https://github.com/TheRealMichaelWang/minima/tree/main/src

This is my build script:

```
@ECHO OFF
occ /c builtins.c
occ /c chunk.c
occ /c collection.c
occ /c compiler.c
occ /c debug.c
occ /c garbage.c
occ /c globals.c
occ /c hash.c
occ /c io.c
occ /c machine.c
occ /c object.c
occ /c opcodes.c
occ /c operators.c
occ /c record.c
occ /c scanner.c
occ /c source.c
occ /c statments.c
occ /c value.c
occ /c variable.c
```

Failed at debug.c and io.c. I checked io.c and found error is at line 124 regarding PRIu64.

**Comment by 3oheicrw** [comment]
I changed build script to compile with C11 (occ /1 /c) but still errors about PRIu64.

**Comment by LADSoft** [comment]
@chuggafan - it seems like that according to the C spec, there shouldn't be any ifdef whatsoever around the inttype macros in inttypes.h.    Apparently __STDC_FORMAT_MACROS_ was purely a C++ thing...   that got deprecated in C++11.    Description of inntypes.h in the C spec doesn't seem to have it....   do you concur we should just remove that ifdef?

**Comment by chuggafan** [comment]
Yes, it's something that should be removed unless we want to support C++98.

**Comment by LADSoft** [comment]
I did fix this at the same time I fixed #318.    The fix got attributed to the wrong issue, here it is: 29de9853b6508bdf755a2089b160173ce2eafec2
